mapfishapp/src/test/java/org/georchestra/mapfishapp/ws/AddonControllerTest.java

```diff
@@ -20,6 +22,10 @@
 
 public class AddonControllerTest {
 
+	public static @BeforeClass void init(){
```

Comment:
Where does theses tab chars come from ?  Don't know whether we have to use tab or whitespace.